.travis.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+function build_container(){
+    docker build -t travis/image-inspector-base .
+    docker build -t travis/image-inspector -f Dockerfile.travis .
+}
+
+function run_tests(){
+  docker run --rm --privileged \
+          -v /var/run/docker.sock:/var/run/docker.sock \
+          --entrypoint make \
+          travis/image-inspector verify test-unit
```

Comment:
@ilackarms line breaks and indentation here seems a little inconsistent, I'd pick one of these:

```bash
docker run \
  --rm \
  --privileged \
  -v /var/run/docker.sock:/var/run/docker.sock \
...
```

```bash
docker run --rm --privileged \
           -v /var/run/docker.sock:/var/run/docker.sock \
           --entrypoint make \
           ...
```